test/lib/send_with_us/api_request_test.rb

```diff
@@ -86,6 +86,21 @@ def test_send_with_with_attachment
     assert_instance_of( Net::HTTPOK, result )
   end
 
+  def test_send_with_with_file
```

Comment:
I know this is a little strange, but can change this to `test_send_email_with_file`. It looks like some tests need to be renamed since they no longer test the **send_with** method.